app/views/homepage/_references.html.erb

```diff
@@ -0,0 +1,101 @@
+<%
+  references = [
+    {
+      title: "Pipeline",
+      links: [
+        {
+          title: "Step types",
+          link: "/docs/pipelines/command-step"
+        },
+        {
+          title: "Conditionals",
+          link: "/docs/pipelines/conditionals"
+        },
+        {
+          title: "Environment variables",
+          link: "/docs/pipelines/environment-variables"
+        },
+        {
+          title: "Emojis",
+          link: "https://github.com/buildkite/emojis#emoji-reference",
+          is_external: true
+        }
+      ]
+    },
+    {
+      title: "Agent",
+      links: [
+        {
+          title: "Configuration settings",
+          link: "/docs/agent/v3/configuration"
+        },
+        {
+          title: "Lifecycle hooks",
+          link: "/docs/agent/v3/hooks"
+        },
+        {
+          title: "CLI commands",
+          link: "/docs/agent/v3/cli-start"
```

Comment:
If you merge main, you should be able to link to the overview: `/docs/agent/v3/cli-reference`